src/Kanbas/Courses/Quizzes/QuizList.tsx

```diff
@@ -1,5 +1,197 @@
+import { useEffect, useState } from "react";
+import "./index.css";
+import {
+    FaEllipsisV,
+    FaCheckCircle,
+    FaGripVertical,
+    FaCaretDown,
+    FaPlus,
+    FaRegCheckCircle,
+} from "react-icons/fa";
+import { useParams } from "react-router";
+import { KanbasState, Module } from "../../store";
+import { useDispatch, useSelector } from "react-redux";
+import {
+    addModule,
+    deleteModule,
+    setModule,
+    updateModule,
+} from "../Modules/modulesReducer";
+import { Link } from "react-router-dom";
+import { formatDate } from "../Assignments";
+import axios from "axios";
+// impport quiz data from mock data json file
+import { quizzes } from "../../Database"
+const API_BASE = process.env.REACT_APP_API_BASE;
+const QUIZ_API = `${API_BASE}/api`;
+
+
+interface Question {
+    _id: string;
+    type: string;
+    points: number;
+    description: string;
+    options: (string | number)[];
+    answers: (string | number)[];

```

Comment:
i might be wrong/misunderstanding how this interface is being, but i think these are both stored as `string[]` in our db and not `(string | number)[]`

Reply:
ooo yep, good catch! That was from my old mock data I'll fix that right away!